scripts/init-compliance-tool-for-environment.sh

```diff
@@ -1,57 +1,85 @@
 #!/usr/bin/env bash
 set -e
 
-if [ ! $# -eq 2 ] ; then
-	echo "this script requires the environment and a persistentId to be passed in as arguments. e.g."
-	echo "$ ./init-compliance-tool.sh <environment> <persistentId>"
+if [ ! $# -eq 3 ] ; then
+	echo "this script requires the environment and a persistentId to be passed in as arguments, along with either the '--matching' flag or the name of a file containing matching dataset json. e.g."
+	echo "$ ./init-compliance-tool.sh <environment> <persistentId> --matching"
 	echo "please try again, currently allowed environments are 'local' and 'dev'"
 	exit 1
 fi
 
 echo "setting persistentID as $2"
 PID=$2
 
+COMPLIANCE_TOOL_INIT_URL_SLUG="relying-party-service-test-run"
+if [ ${3-""} == "--matching" ] ; then
+  COMPLIANCE_TOOL_INIT_URL_SLUG="service-test-data"
+fi
 
 case $1 in
   	'local')
 		echo "configuring compliance tool for Relying Party running on localhost:3200"
 		SERVICE_ENTITY_ID="http://verify-service-provider-local"
 		ASSERTION_CONSUMER_SERVICE_URL="http://localhost:3200/verify/response"
+		COMPLIANCE_TOOL_INIT_URL="http://localhost:50270/$COMPLIANCE_TOOL_INIT_URL_SLUG"
+    MATCHING_SERVICE_ENTITY_ID="http://www.test-rp-ms.gov.uk/SAML2/MD"
+    MATCHING_SERVICE_SIGNING_KEY=`cat ../test-keys-and-certs/msa-signing-key.pem`
```

Comment:
Should we be using the same `MATCHING_SERVICE_SIGNING_KEY` specified for the `dev` environment or do we need to add this key to the `test-keys-and-certs` directory? I seem to remember there was some discussion around this.

Reply:
@karlbaker02 we will change this to use the same `MATCHING_SERVICE_SIGNING_KEY` as the dev environment. This eliminates the need for generating that `msa-signing-key.pem` file.